src/BaGetter.Core/Configuration/BaGetterOptions.cs

```diff
@@ -58,4 +64,6 @@ public class BaGetterOptions
     public HealthCheckOptions HealthCheck { get; set; }
 
     public StatisticsOptions Statistics { get; set; }
+
+    public AuthenticationOptions[] Authentication { get; set; }
```

Comment:
I'd prefer this to be an `AuthenticationOptions` object that contains a `Credential[] Credentials` or something, I think that would make it easier to modify later with less breaking changes.